ext/drx/drx.c

```diff
@@ -2523,44 +2523,56 @@ drx_expand_scatter_gather(void *drcontext, instrlist_t *bb, OUT bool *expanded)
 typedef struct _drx_state_machine_params_t {
     byte *pc;
     byte *prev_pc;
+    /* state machine's state. */
```

Comment:
style nit: either capitalized and punctuated, or neither

Reply:
Fixed.